### ISSUE TITLE
ComputerCraft Turtle peripheral integration

### ISSUE THREAD

**Issue by Merith-TK** [CammiePone/Cam-s-Wearable-Backpacks#22]
Basically just requestion you make the backpacks wearable by turtles so a turtle can use it for storage!

**Comment by Merith-TK** [comment]
you should be able to just expose the backpack's "inventory" to the turtle, and expose the "crafting" api thats already in the game/mod for crafting tables to the turtle and then its a two in one, inventory expansion and making it a crafting turtle!

**Comment by CammiePone** [comment]
It usually isnt that easy, but it would be really cute to see the turtles wearing the backpacks so Im considering it

**Comment by Merith-TK** [comment]
yeah! Toad-Dev has an example peripheral you can look at for registering a block as a thing, 
only one problem about it is that it can be on the left or right of the turtle only, which sucks...

https://github.com/Toad-Dev/cc-peripheral-test

since the dev of CC-Tweaked left mc modding, we might be able to talk to the community about adding a third peripheral slot to turtles...

**Comment by drakosoldier298** [comment]
Merith is one of the devs for cc fabiric btw.

**Comment by Merith-TK** [comment]
I mostly do management stuff while ToadDev does most of the work as of late, I used to be going commit by commit in updating it to be as close to the forge version as possible back in CC:T 1.95, gods that was slow, then people found it and started contributing, hell even Squid (CC:T dev) contributed a bit to the project, mainly in adding the CC:T linting checkers and stuffs so merging diffs from CC:T would be easier!

**Comment by CammiePone** [comment]
Oh shit, okay. Didnt know that. Then maybe it is that easy 👀

**Comment by Merith-TK** [comment]
okay if you want the backpack to be on the back your gonna have to model it so on turtles they are rendered at the back, but with a indicator as to what side slot its equipt on
